Cubical/Foundations/Family.agda

```diff
@@ -0,0 +1,54 @@
+{-# OPTIONS --cubical --safe #-}
+
+module Cubical.Foundations.Family where
+
+open import Cubical.Foundations.Logic
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Equiv
+
+private
+  variable
+    ℓ₀ ℓ₁ ℓ₂ : Level
+
+Fam : (ℓ₀ : Level) → Type ℓ₁ → Type (ℓ-max (ℓ-suc ℓ₀) ℓ₁)
+Fam ℓ A = Σ (Set ℓ) (λ I → I → A)
```

Comment:
Please use the nice notation for Sigma types and rename all uses of Set to Type. Also, please don't use `I` for variables as we use it for the interval

Reply:
I guess you can use Fam in https://github.com/agda/cubical/blob/master/Cubical/Functions/Fibration.agda#L60